api/types/signal.go

```diff
@@ -0,0 +1,7 @@
+package apitypes
+
+// SignalBody encapsulates the payload for a call to a plugin's Signal function
+type SignalBody struct {
```

Comment:
I chose this format because it's easier to extend it if `Signal` ever supports more arguments.